launcher/OptionsGraphicsDlg.cpp

```diff
@@ -90,6 +92,7 @@ void OptionsGraphicsDlg::InitToolTip()
     m_tool_tip.AddTool(GetDlgItem(IDC_HIGH_SCANNER_RES_CHECK), "Increase scanner resolution (used by Rail Gun, Rocket Launcher and Fusion Launcher)");
     m_tool_tip.AddTool(GetDlgItem(IDC_HIGH_MON_RES_CHECK), "Increase monitors and mirrors resolution");
     m_tool_tip.AddTool(GetDlgItem(IDC_TRUE_COLOR_TEXTURES_CHECK), "Increase texture color depth - especially visible for lightmaps and shadows");
+    m_tool_tip.AddTool(GetDlgItem(IDC_UNCLAMP_LIGHTMAPS_CHECK), "Unclamp lightmap rendering colour range (allows for fully dark lighting on geometry)");
```

Comment:
color, not colour, let's keep it consistent (US English)

Reply:
Fixed in latest commit.